services/cards-consultation/src/main/java/org/opfab/cards/consultation/configuration/webflux/CardSubscriptionRoutesConfig.java

```diff
@@ -92,6 +94,17 @@ private HandlerFunction<ServerResponse> cardSubscriptionOptionsRoute() {
         return request -> ok().build();
     }
 
+    /**
+     * Card Subscription DELETE route
+     * @return
+     */
```

Comment:
unneeded comment , the method name already say it

Reply:
Done.